lint/linter.py

```diff
@@ -921,9 +925,19 @@ def lint(self, code, hit_time, settings):
         - If the view has been modified since the original hit_time, stop.
         - Parse the linter output with the regex.
         """
+        # We 'name' our threads, for logging purposes.
+        threading.current_thread().name = task_name
```

Comment:
I believe this should be happening in `execute_lint_task` instead. `Linter.lint` does not need to know or care about the backend being used and how tasks are scheduled.

Reply:
Yeah, that would be better